app/resources/results.html.template

```diff
@@ -46,37 +46,33 @@
       // third party search API directly here, to workaround AJAX's same
       // origin policy, and for authorization.
       function perform_third_party_search(id) {
-        fetch(repo_url + '/third_party_search', {
-          method: 'POST',
-          body: JSON.stringify({
-            query: query,
-            lang: lang,
-            query_type: query_type,
-            search_engine_id: id,
-          }),
-          headers: {
-            'Content-Type': 'application/json',
-          },
-        })
-        .then(r => r.json())
-        .then(function(r) {
-          console_log('Third-party search response: ', r);
-          if (r.status == 'OK') {
-            if (typeof(r.url) != 'string') {
-              console_error(
+        var req = new XMLHttpRequest();
+        const url = (
```

Comment:
May be better to avoid using `const` as well because it is part of ES6.

Reply:
Oh, I hadn't realized that. Changed to var.